### PR TITLE
Show dd cards when ddeploy flag is set.

### LINKED ISSUE #3087: Entity details cards has dt link, not dd

### PR DESCRIPTION
**Pull request by hatched**
Fixes #3087

### CODE REVIEW

**Comment by hatched** [comment]
#### To QA

Load the GUI with the `ddeploy` flag set and then check that the card in the entity details has `data-dd`.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7898/

File: jujugui/static/gui/src/app/components/entity-details/content/content.js [hunk 1]
```diff
@@ -555,9 +555,10 @@ class EntityContent extends React.Component {
     const entity = entityModel.toEntity();
     const storeId = entity.type === 'charm' ?
       entity.storeId : entity.id.split('cs:').join('');
+    const dataDD = this.props.flags.ddeploy ? 'data-dd' : '';
```
Comment (mitechie) [inline jujugui/static/gui/src/app/components/entity-details/content/content.js]: data-dd? is there something more descriptive?

Comment (hatched) [inline jujugui/static/gui/src/app/components/entity-details/content/content.js]: this is what tells the cards to use the dd link. If you would like that changed a new version of the cards library will need to be created.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7899/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7900/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7902/

File: jujugui/static/gui/src/app/components/entity-details/content/test-content.js [hunk 2]
```diff
@@ -196,6 +196,28 @@ describe('EntityContent', function() {
...
+        pluralize={sinon.stub()}
+        renderMarkdown={sinon.stub()}
+        scrollCharmbrowser={sinon.stub()}
+        showTerms={sinon.stub()}
+        staticURL="http://example.com" />);
+    const card = output.props.children[2].props.children.props.children[1].
+      props.children[5].props.children[4];
+    const expected = (
+      <div className="juju-card" data-dd data-id="trusty/django-123"></div>);
+    expect(card).toEqualJSX(expected);
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/entity-details/content/test-content.js]: Should this check the example line too?

Comment (hatched) [inline jujugui/static/gui/src/app/components/entity-details/content/test-content.js]: Yep, added

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7903/

**Comment by hatched** [comment]
Thanks all :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**